HWindex.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<!--suppress ALL -->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My homework4</title>
+</head>
+<body>
+<head>
+    <h1>Form</h1>
+</head>
+<form action="">
+    <label for="name">First name</label>
+    <input type="text" id="form" name="form"><br><br>
+    <label for="email">Email</label>
+    <input type="email" id="email" name="email"><br><br>
+    <label for="lname">Last name</label>
```

Comment:
please provide more relevant name 
lname - is not understandable 
last-name - better naming